frontend/javascripts/admin/dataset/dataset_upload_view.tsx

```diff
@@ -995,6 +1005,19 @@ class DatasetUploadView extends React.Component<PropsWithFormAndRouter, State> {
                     return archives.length === 0 || archives.length === files.length;
                   }, "Archives cannot be mixed with other files."),
                 },
+                {
+                  validator: syncValidator(
+                    (files: FileWithPath[]) => {
+                      const fileSize = getFileSize(files);
+                      return getLeftOverStorageBytes(this.props.organization) > fileSize;
```

Comment:
Nitpicky, but I'd say this is more correct :)

```suggestion
                      return getLeftOverStorageBytes(this.props.organization) >= fileSize;
```